src/rgw/rgw_group.h

```diff
@@ -0,0 +1,132 @@
+#ifndef CEPH_RGW_GROUP_H
+#define CEPH_RGW_GROUP_H
```

Comment:
@cbodley : I don't see any rgw header file having forward declaration of CephContext and RGWRados and also none of them include include/encoding.h. I think some header file does all three which gets included by a common rgw header file. Do I still need to include these two?